tests/_utils/start_tidb_cluster_impl

```diff
@@ -213,13 +213,6 @@ while ! mysql -uroot -h${DOWN_TIDB_HOST} -P${DOWN_TIDB_PORT} --default-character
     sleep 2
 done
 
-# TODO: remove this after TiCDC supports TiDB clustered index
-run_sql "select tidb_version();set @@global.tidb_enable_clustered_index=0;" ${UP_TIDB_HOST} ${UP_TIDB_PORT} || echo "tidb_enable_clustered_index not supports"
```

Comment:
Please also remove the similar patch in file `tests/_utils/start_tls_tidb_cluster_impl`